language-plutus-core/src/Language/PlutusCore/Core/Type.hs

```diff
@@ -116,8 +115,6 @@ data Term tyname name uni ann
     | Error ann (Type tyname uni ann)
     deriving (Show, Functor, Generic, NFData, Lift, Hashable)
 
-type Value = Term
```

Comment:
We no longer need this, since the machines have their own types of values now.  It was always a bit annoying that we had values but didn't have a separate type to enforce allowing only values in certain contexts.